src/main/java/chess/domain/piece/Rook.java

```diff
@@ -0,0 +1,27 @@
+package chess.domain.piece;
+
+import chess.domain.position.Direction;
+import java.util.List;
+
+public class Rook extends Piece {
+
+    private static final int MAX_UNIT_MOVE = 7;
+    private static final List<Direction> ROOK_DIRECTION = List.of(
+            Direction.POSITIVE_FILE_SAME_RANK,
+            Direction.NEGATIVE_FILE_SAME_RANK,
+            Direction.SAME_FILE_POSITIVE_RANK,
+            Direction.SAME_FILE_NEGATIVE_RANK
+    );
+
+    public Rook(Color color) {
+        super(PieceType.ROOK,
+                color,
+                ROOK_DIRECTION
+        );
```

Comment:
```suggestion
        super(PieceType.ROOK, color, ROOK_DIRECTION);
```
과도한 줄바꿈은 가독성에 오히려 해가 되어요. refactoring에서 길이로 제약사항이 걸리지 않는 이상 줄바꿈은 지양하는 것이 좋습니다.